classes/output/tabs.php

```diff
@@ -73,6 +72,12 @@ public function export_for_template(\renderer_base $output): array {
                     'title' => get_string('report_unfinished', 'tool_excimer'),
                     'text' => get_string('unfinished', 'tool_excimer'),
                 ],
+                [
+                    'id' => 'page_groups',
+                    'link' => [[ 'link' => new \moodle_url('/admin/tool/excimer/page_groups.php') ]],
```

Comment:
this might end up merged into the existing tabs rather than a new tab, we can refine this toward the end